src/theme/Footer/styles.module.css

```diff
@@ -1,41 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-.footerLogoLink {
-  opacity: 0.5;
-  transition: opacity 0.15s ease-in-out;
-}
-
-.footerLogoLink:hover {
-  opacity: 1;
-}
-
-/* Osano Styles  */
-
-.osano-cm-widget svg {
-  display: none;
-}
-.osano-cm-widget svg {
-  display: none;
-}
-.osano-cm-widget:focus,
-.osano-cm-widget:hover {
-  opacity: 1;
-  transform: none;
-}
-.osano-cm-widget:active {
-  transform: translateY(1px);
-}
-.osano-cm-link {
-  color: #002f66 !important;
-}
-.osano-cm-close {
-  background: transparent !important;
-}
-.osano-cm-view--type_consent .osano-cm-list-item:nth-child(5) {
-  display: none;
-}
```

Comment:
These Osano styles are for the consent popup, but the ones that are actually used are [in src/custom.css](https://github.com/camunda/camunda-platform-docs/blob/d689488d5b7aef3252c0792b39505ece4e220057/src/css/custom.css#L81).

Reply:
Visual proof that there is no _regression_ due to removing these styles: 

<img width="2043" alt="image" src="https://user-images.githubusercontent.com/1627089/182718783-27f8b3ce-2c46-4a82-9134-bc9fd53070ab.png">

I say _regression_ because @akeller raised concerns that the styles in the code do not match the styles given to us by marketing, which I will follow up on (and will not block the merge of this PR).

Reply:
This was my 🗑️, but I followed up with you on Slack about some discrepancies and to check in with the web marketing team. This doesn't block this PR.